<?xml version="1.0"?>
<launch>
    <arg name="src_frame_name" default="world"/>
    <arg name="object_name" default="HV8"/>
    <arg name="object_count" default="7"/>
    
    <node name="model_bara" pkg="tf_publish" type="model_tf_publisher_bara"  output="screen">
        <param name="src_frame_name" value="$(arg src_frame_name)"/>
        <param name="object_name" value="$(arg object_name)"/>
        <param name="object_count" value="$(arg object_count)"/>
        
    </node>

</launch>